toolchain/musl/Makefile

```diff
@@ -11,6 +11,10 @@ MUSL_MAKEOPTS = -C $(HOST_BUILD_DIR) \
 	DESTDIR="$(TOOLCHAIN_DIR)/" \
 	LIBCC="$(subst libgcc.a,libgcc_initial.a,$(shell $(TARGET_CC) -print-libgcc-file-name))"
 
+ifeq ($(CRYPTO_SIZE_HACK),y)
+TARGET_CFLAGS += -DSIZE_HACK=y
```

Comment:
From source: `#ifndef SIZE_HACK`, I think this line should be `TARGET_CFLAGS += -DSIZE_HACK`

Reply:
Fixed, thanks